Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -64,12 +64,13 @@ class DisplayHymnViewModel: ObservableObject {
                     guard let self = self else { return }
                     guard let hymn = hymn else { return }
 
-                    let title: Title
+                    var title: Title
```

Comment:
I think this can still be a let

Reply:
It complained later when I did the append since let made it immutable.